js/app.js

```diff
@@ -6,20 +6,29 @@ $(document).ready(function(){
 
   function illuminateRed(){
     console.log("calling function")
+    clearLights();
     $('#stopLight').css('background-color', 'red');
+
     // $('#slowLight').css('background-color', 'black');
     // $('#goLight').css('background-color', 'black');
   }
 
   function illuminateYellow(){
     console.log("calling function")
+    clearLights();
     $('#slowLight').css('background-color', 'yellow');
   }
 
   function illuminateGreen(){
     console.log("calling function")
+    clearLights();
     $('#goLight').css('background-color', 'green');
   }
 
+  function clearLights(){
+    $('#stopLight').css('background-color', 'black');
+    $('#slowLight').css('background-color', 'black');
+    $('#goLight').css('background-color', 'black');
+  }
```

Comment:
@lswatson this function works great, but did notice that all three lights share the Class "bulb" and well instead of what you have you can also use the class like this: $('.bulb').css('background-color', 'black');   :thumbsup:

Reply:
Yes that would be a better way because it requires less code. Thank you @axjimenez89